<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>aziz</class>
 <widget class="QMainWindow" name="aziz">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>973</width>
    <height>509</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>aziz</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 255, 128);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>891</width>
      <height>471</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 255, 255);</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Gestion de Vehicule</string>
     </attribute>
     <widget class="QComboBox" name="id_employee1">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>260</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="destination">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>220</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>340</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="text">
       <string>DATE</string>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>260</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="text">
       <string>ID EMPLOYEE</string>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QTableView" name="data">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>120</y>
        <width>411</width>
        <height>281</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>180</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="text">
       <string>ID VEHICULE</string>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>10</y>
        <width>47</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Par:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_supprimer_3">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>20</y>
        <width>81</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Modifier</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>10</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
      <property name="editable">
       <bool>false</bool>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="insertPolicy">
       <enum>QComboBox::NoInsert</enum>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QComboBox::AdjustToContentsOnFirstShow</enum>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
      <item>
       <property name="text">
        <string>IDVEHCULE</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>DESTINATION</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ID_EMPLOYEE</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ID_STOCK</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="pb_ajouter">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>20</y>
        <width>81</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Ajouter</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="id_vehicule2">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>180</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QFrame" name="frame">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>120</y>
        <width>301</width>
        <height>291</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(233,248,255);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="lineWidth">
       <number>2</number>
      </property>
      <property name="midLineWidth">
       <number>0</number>
      </property>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>180</y>
         <width>101</width>
         <height>20</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <property name="text">
        <string>ID STOCK</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </widget>
     <widget class="QLineEdit" name="lineEdit_id_6">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>211</width>
        <height>21</height>
       </rect>
      </property>
      <property name="inputMethodHints">
       <set>Qt::ImhNoPredictiveText</set>
      </property>
      <property name="placeholderText">
       <string>Recherche...</string>
      </property>
      <property name="clearButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_supprimer_4">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>20</y>
        <width>81</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Supprimer</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="DATE">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>340</y>
        <width>110</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="id_tri">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>40</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>IDVEHICULE</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>DESTINATION</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>220</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="text">
       <string>DESTINATION</string>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QComboBox" name="id_stock">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>300</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>40</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Tri√© Par:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>70</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>PrintPDF</string>
      </property>
     </widget>
     <zorder>frame</zorder>
     <zorder>id_employee1</zorder>
     <zorder>destination</zorder>
     <zorder>label_2</zorder>
     <zorder>label</zorder>
     <zorder>data</zorder>
     <zorder>label_3</zorder>
     <zorder>label_5</zorder>
     <zorder>pb_supprimer_3</zorder>
     <zorder>comboBox</zorder>
     <zorder>pb_ajouter</zorder>
     <zorder>id_vehicule2</zorder>
     <zorder>lineEdit_id_6</zorder>
     <zorder>pb_supprimer_4</zorder>
     <zorder>DATE</zorder>
     <zorder>id_tri</zorder>
     <zorder>label_4</zorder>
     <zorder>id_stock</zorder>
     <zorder>label_17</zorder>
     <zorder>pushButton_2</zorder>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Gestion de stock</string>
     </attribute>
     <widget class="QComboBox" name="id_vehicule">
      <property name="geometry">
       <rect>
        <x>710</x>
        <y>320</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="date_ajout">
      <property name="geometry">
       <rect>
        <x>710</x>
        <y>280</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>360</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="text">
       <string>Date de livraison</string>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>320</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="text">
       <string>ID VEHICULE</string>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QTableView" name="data_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>150</y>
        <width>411</width>
        <height>271</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>240</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="text">
       <string>ID STOCKS</string>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>10</y>
        <width>47</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Par:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_supprimer_5">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>20</y>
        <width>81</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Modifier</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_2">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>10</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
      <property name="editable">
       <bool>false</bool>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="insertPolicy">
       <enum>QComboBox::NoInsert</enum>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QComboBox::AdjustToContentsOnFirstShow</enum>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
      <item>
       <property name="text">
        <string>ID</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ID_STOCKS</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>DATE_AJOUT</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ID_MATCH</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>DATE_DE_LIVRAISON</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="pb_ajouter_2">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>20</y>
        <width>81</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Ajouter</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="id_stocks">
      <property name="geometry">
       <rect>
        <x>710</x>
        <y>240</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QFrame" name="frame_2">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>140</y>
        <width>301</width>
        <height>291</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(233,248,255);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="lineWidth">
       <number>2</number>
      </property>
      <property name="midLineWidth">
       <number>0</number>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_id_7">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>211</width>
        <height>21</height>
       </rect>
      </property>
      <property name="inputMethodHints">
       <set>Qt::ImhNoPredictiveText</set>
      </property>
      <property name="placeholderText">
       <string>Recherche...</string>
      </property>
      <property name="clearButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_supprimer_6">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>20</y>
        <width>81</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Supprimer</string>
      </property>
     </widget>
     <widget class="QComboBox" name="id_tri2">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>40</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>ID_EMPLOYEE</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ID_STOCKS</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>DATE_AJOUT</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ID_VEHICULE</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>DATE_DE_LIVRAISON</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>280</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="text">
       <string>DATE D'AJOUT</string>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>200</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="text">
       <string>ID EMPLOYEE</string>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="id_employee">
      <property name="geometry">
       <rect>
        <x>710</x>
        <y>200</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="date_de_livraison">
      <property name="geometry">
       <rect>
        <x>710</x>
        <y>360</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>40</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Tri√© Par:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>70</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>PrintPDF</string>
      </property>
     </widget>
     <zorder>frame_2</zorder>
     <zorder>id_vehicule</zorder>
     <zorder>date_ajout</zorder>
     <zorder>label_6</zorder>
     <zorder>label_7</zorder>
     <zorder>data_2</zorder>
     <zorder>label_8</zorder>
     <zorder>label_9</zorder>
     <zorder>pb_supprimer_5</zorder>
     <zorder>comboBox_2</zorder>
     <zorder>pb_ajouter_2</zorder>
     <zorder>id_stocks</zorder>
     <zorder>lineEdit_id_7</zorder>
     <zorder>pb_supprimer_6</zorder>
     <zorder>id_tri2</zorder>
     <zorder>label_10</zorder>
     <zorder>label_11</zorder>
     <zorder>id_employee</zorder>
     <zorder>date_de_livraison</zorder>
     <zorder>label_18</zorder>
     <zorder>pushButton_3</zorder>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>973</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
